integrated_channel_plugin_sapsf/apps.py

```diff
@@ -0,0 +1,13 @@
+"""
+Enterprise Integrated Channel SAP SuccessFactors Django application initialization.
+"""
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class IntegratedChannelPluginSapsfConfig(AppConfig):
```

Comment:
I wonder if there should just be an `integrated_channels` app in the edx-enterprise Django project. You could then have separate directories for each `IntegratedChannel` implementation.